python3-gceimgutils.spec

```diff
@@ -60,8 +67,9 @@ gzip %{buildroot}/%{_mandir}/man1/*
 %doc README.md
 %license LICENSE
 %{_mandir}/man*/*
-%dir %{python3_sitelib}/gceimgutils
-%{python3_sitelib}/*
+%dir %{python_sitelib}/gceimgutils
+%{python_sitelib}/*
```

Comment:
the "binary" needs an alternative setup, see https://github.com/SUSE-Enceladus/ec2imgutils/blob/master/python-ec2imgutils.spec#L146